File: pinyougou/pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;
/**
 * 地址接口
 * @author Administrator
 *
 */
import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbAddress;
import com.pinyougou.pojo.TbBrand;

import entry.PageResult;



public interface BrandService {
  
    public List<TbBrand> findAll();
    
   public  PageResult  findPage(int pageNum,int pageSize);
   
   public  PageResult  findPage01(TbBrand tbBrand ,int pageNum,int pageSize);
   
   public void  save(TbBrand tbBrand);
   
   public TbBrand findone(Long id);
   
   public void update(TbBrand tbBrand );
   
   public void deleteById(Long[] ids);
   
   public List<Map> selectOptionList();
}

File: pinyougou/pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function($http){
    	
    	this.findAll=function(){
    		return  $http.get('../brand/findAll.do');
       	}
       	
    	
       	this.findPage=function(page,size){
    		return $http.get("../brand/findPage.do?page="+page+"&size="+size);
    	}
       	
       this.findSearch=function(page,size,searchEntity){
    	   return $http.post('../brand/search.do?page='+page+'&size='+size,searchEntity);
       }
       
      this.findOne=function(id){
    	 
    	 return $http.post("../brand/findOne.do?id="+id);
      }
      
     this.del=function(selectIds){
    	 return $http.get("../brand/delete.do?ids="+selectIds);
     }
     
     this.save=function(entry){
    	 return $http.post("../brand/save.do",entry);
     }
     
     this.update=function(entry){
    	 return $http.post("../brand/update.do",entry);
     }
     
     this.selectOptionList=function(){
    	 return $http.post("../brand/selectOptionList.do"); 
     }
       	
    } );
    
    